backend/apid/graphql/event_filter.go

```diff
@@ -0,0 +1,46 @@
+package graphql
+
+import (
+	v2 "github.com/sensu/sensu-go/api/core/v2"
+	"github.com/sensu/sensu-go/backend/apid/graphql/globalid"
+	"github.com/sensu/sensu-go/backend/apid/graphql/schema"
+	"github.com/sensu/sensu-go/graphql"
+	"github.com/sensu/sensu-go/types"
+	"github.com/sensu/sensu-go/util/strings"
+)
+
+var _ schema.EventFilterFieldResolvers = (*eventFilterImpl)(nil)
+
+//
+// Implement EventFilterFieldResolvers
+//
+
+type eventFilterImpl struct {
+	schema.EventFilterAliases
+}
```

Comment:
Can you double check if the the `type` field is working? it might conflict with the enum.

Reply:
Yeah -- you mean the `action` field, and yes -- we'll need to write a custom resolver for this.